crates/bevy_winit/src/lib.rs

```diff
@@ -7,6 +7,7 @@ use bevy_input::{
     mouse::{MouseButtonInput, MouseMotion, MouseScrollUnit, MouseWheel},
     touch::TouchInput,
 };
+pub use winit::event_loop::ControlFlow;
```

Comment:
This is the only type reexported from winit in Bevy. I would prefer if it was mapped to a Bevy struct.